internal/kubernetes/operator/install.go

```diff
@@ -0,0 +1,311 @@
+// Copyright 2023 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package operator
+
+import (
+	"context"
+	"errors"
+	"fmt"
+
+	akov1 "github.com/mongodb/mongodb-atlas-kubernetes/pkg/api/v1"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/mongodb/mongodb-atlas-cli/internal/kubernetes"
+	"github.com/mongodb/mongodb-atlas-cli/internal/kubernetes/operator/features"
+	"github.com/mongodb/mongodb-atlas-cli/internal/kubernetes/operator/resources"
+	"github.com/mongodb/mongodb-atlas-cli/internal/store"
```

Comment:
We are transitioning to a auto generated sdk, we would need you to start using that 
```suggestion
	store "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas"
```

Reply:
@gssbzn But I'm relying on interfaces defined on the store package too, it doesn't make sense to point only/direct to atlas package

Reply:
"github.com/mongodb/mongodb-atlas-cli/internal/store" is now suppose to be used by mongocli only while "github.com/mongodb/mongodb-atlas-cli/internal/store/atlas" is what atlas cli should use, if you comit to migrating these new commands to the new SDK soonish I'm ok to leave as is as we are about to wrap the migration processes to the new SDK and from now on the manual SDK will enter maintenance mode

Reply:
okay, I understand that. Our store interfaces were defined in the `store` package. I'll create a ticket to move them to the `atlas` pkg and import it as suggested.